packages/extension/src/newtab/App.tsx

```diff
@@ -134,12 +146,29 @@ function InternalApp(): ReactElement {
     }
   }, [contentScriptGranted]);
 
+  useEffect(() => {
+    if (shouldShowLogin) {
+      return;
+    }
+
+    getCache(INSTALLATION_STORAGE_KEY).then((value) => {
+      if (value) {
+        logEvent({ event_name: LogEvent.ShowNewTabPermission });
```

Comment:
Let's add the event to the component itself since this can be true while the user isn't part of the experiment